portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,107 @@
-  empty_state: There are no accounts
-  empty_state_description: Message of the Accounts table empty state
-  col_group: Group/org.
-  col_group_description: Title of the Group/org. column
-  col_admin: Admin
-  col_admin_description: Title of the Admin column
-  col_signup: Signup date
-  col_signup_description: Title of the Signup date column
-  col_apps: Apps
-  col_apps_description: Title of the Apps column
-  col_state: State
-  col_state_description: Title of the State column
+# Account settings > Users
+
+title_page: Accounts | Team members
+title_page_description: Title for the page to manage team members
+
+# Top area
+
+body_title: Team members
+body_title_description: Title for the main area to manage team members
+invite_new_member_button: Invite a new team member
+invite_new_member_button_description: Button link to send invitations to new team members
+
+# Main area
+
+## Tabs
+listing_tab: Listing
+listing_tab_description: Label for the tab that contains the list of all team members, selection criteria and actions that can be applied on them.
+invitations_tab: Invitations
+invitations_tab_description: Label for the tab that contains the list of all invitations, selection criteria and actions that can be applied on them.
+
+## Tab: Listing
+## Toolbar
+#-------# Bulk selector: available in shared.yml
+#-------# Delete button: available in shared.yml
+team_members_filter: Name
+team_members_filter_description: List containing the name criterion to filter team members
+team_members_typeahead: Filter by name #----PROPOSED
+team_members_typeahead_description: Default text to show when the user wants to filter by typing the text
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  empty_state: Currently, there are no team members. Add team members. #----PROPOSED
+  empty_state_description: Message to show when there are no team members to show, and prompt the user to add members
+  name_header: Name
+  name_header_description: Column listing names of team members
+  email_header: Email
+  email_header_description: Column listing the email addresses of team members
+  sso_authentication_heaer: SSO authentication
+  sso_authentication_header_description: Column listing if SSO authentication has been configured for team members
+  sso_authentication_value:
+    yes: Yes
+    not_yet: Not yet
+  sso_authentication_value_description: Container of possible values about users having SSO configured
+  role_header: Role
+  role_header_description: Column listing the roles associated to each team member
+  role_header_value: #----PROPOSED
+    admin_you: Admin (you)
+    admin: Admin
+    member: Member
+  role_header_value_description: Container with possible values of the roles of each team member
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
```

Comment:
In this case, I'll propose an **Actions** column, for accessibility purposes.

Reply:
@porueesq  the contextual row menu triggered by the kebab button is a Patternfly standard. Exposing the actions would mean featuring two more columns in this case, and to keep consistency it would eventually mean many more columns in scenarios where multiple actions are available for a row -- which makes it a non-viable option.

Reply:
@cntlsn @damianpm 
We need an alternative to describe that the last column contains all the possible actions.This is a matter of accessibility. 
A screen reader goes from left to right, and if it finds an empty cell in the header row, it won't report anything. Then, when the reader moves along the next rows, it will find the available actions, hence creating confusion to the user with the reader.
Some alternatives include, but not limited to: 

- an **Actions** header 
- a tooltip describing that the column contains actions to be applied to the relevant row.

Reference: [Accessibility principles](https://www.w3.org/WAI/fundamentals/accessibility-principles/#alternatives)

Reply:
In [patternfly tables](https://www.patternfly.org/v4/documentation/react/components/table#actions) I don't see any option of adding a header or a tooltip for the Actions column.
Maybe we can add a header but hide it with css, but maybe is not a best practice...

Reply:
@porueesq Thank you for raising this, I understand your point.
About the "Actions" header, I think this should be discussed with the PatternFly team. Please refer to PF [design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/lists-and-tables) and [documentation](https://www.patternfly.org/v4/documentation/react/components/table) for tables: no "Actions" column header is ever mentioned, albeit actions and kebab menus being present in some of the examples.
I'm not saying it is wrong, I'm just saying that this change would have consequences in some scenarios. For instance, please have a look at [this mockup](https://marvelapp.com/ab13h96/screen/67417389) where I have added the "Actions" column header after your suggestion. Since both a button and a kebab menu are included in each row, shall we have two "Actions" column headers? It seems a bit too redundant.
My suggestion would be to include an "Actions" column header for columns that include buttons, but not for the ones that include kebab icons.

Sure we can – and probably should – add a tooltip for kebab icons, but this would only partially solve the problem, as tooltips can't be triggered on touchscreen devices. Unless screenreaders will be able to detect tooltips on touchscreen devices, which could be unlikely according to:
* https://inclusive-components.design/tooltips-toggletips/
* https://ux.stackexchange.com/questions/71950/help-tooltips-and-screen-readers

@damianpm WDYT?

Reply:
@cntlsn Yes, to fulfil the guidelines and keep simplicity, I'm with your suggestion:
> My suggestion would be to include an "Actions" column header for columns that include buttons, but not for the ones that include kebab icons.

@damianpm Is this also fine with you?

Reply:
I agree we need to discuss this with PF team, in the meantime we can keep it without the additional header

Reply:
Adding this thread to the agenda of our next standup meeting, to agree on action points.

Reply:
As a temporary measure, we'll stick to the suggestion proposed by @cntlsn
> My suggestion would be to include an "Actions" column header for columns that include buttons, but not for the ones that include kebab icons.